source/features/linkify-labels-on-dashboard.tsx

```diff
@@ -8,9 +8,14 @@ import {wrap} from '../helpers/dom-utils';
 import features from '.';
 
 function init(): void {
-	observe('.js-recent-activity-container :not(a) > .IssueLabel', {
+	const labelClass = [
+		'.js-recent-activity-container :not(a) > .IssueLabel', // Recent activity
+		'.js-all-activity-header + div :not(a) > .IssueLabel' // Newsfeed
+	].join();
+	observe(labelClass, {
 		add(label) {
-			const activity = label.closest('li')!;
+			// A rgh class is not needed since we already check for `not(a)` https://github.com/sindresorhus/refined-github/pull/3625#discussion_r501309853
+			const activity = label.closest('li, div:not([class])')!;
```

Comment:
It's best to explain what it's for since Newsfeed events are much more rare.

```suggestion
			const activity = label.closest('li, div:not([class])')!; // `div` is for the Newsfeed
```

Reply:
👍 Forgot to add the comment back